src/geo/map.js

```diff
@@ -47,12 +45,16 @@ var Map = Model.extend({
     this.layers.bind('change:attribution', this._updateAttributions, this);
 
     if (this._dataviewsCollection) {
-
       // When new dataviews are defined, a new instance of the map needs to be created
       this.listenTo(this._dataviewsCollection, 'add', _.debounce(this._onDataviewAdded.bind(this), 10));
     }
   },
 
+  instantiateMap: function () {
+    this._initBinds();
```

Comment:
Yes, if not, when the layers collection is generated and single layers are being added will fire as many `reload` methods as layers has the final map. That means `X` map instantiations.